tests/test.yml

```diff
@@ -8,7 +8,7 @@
 
     - name: Install virtualenv
```

Comment:
Ist the name still correct after this change?

Reply:
No this does not work at the moment